packages/graphback-runtime-mongodb/src/utils/createIndexes.ts

```diff
@@ -85,4 +95,20 @@ export function getRelationIndex(field: GraphQLField<any, any>): IndexSpecificat
   } else {
     return undefined;
   }
+}
+
+export function isCustomIdField(field: GraphQLField<any, any>): IndexSpecification {
+  const indexMetadata: any = parseMetadata('id', field.description);
+  if (indexMetadata === true) {
+    const indexSpec: IndexSpecification = {
+      key: {
+        [field.name]: 1
```

Comment:
How will this work with 
```
""" 
@model
"""
type SomeType{
 """ 
 @id
 @db.name: 'primary_key_colulm'
 """ 
  key: String!
}
```

Reply:
We don't have renaming fields in Mongo yet, but I think we probably need `id:ID!`

Reply:
Oh sure, then this comment can be safely ignored.

Reply:
I do think a custom primary key would be worthwhile looking into instead of just indexing it, so I will try and open an issue for this after a bit of research, Thanks for pointing it out :smile:

Reply:
Amazing thanks for the pro-activity.

Reply:
Hmm.. reminded me that we need to decide what we do with _id.